cpp/include/cudf/io/csv.hpp

```diff
@@ -106,6 +106,8 @@ class csv_reader_options {
   char _quotechar = '"';
   // Whether a quote inside a value is double-quoted
   bool _doublequote = true;
+  // Detect quotes even if surrounded by spaces e.g. `   "data"   `
+  bool _updated_quotes_detection = false;
```

Comment:
I updated the name to `_detect_whitespace_around_quotes`. I chose this name because this flag has no effect on its own. Its purpose is to update how double quotes are detected.

I updated the docs accordingly. And added a note that it has no effect if quotes are kept.